review_practice/react-app/src/containers/RoomList/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useContext } from 'react';
+import { useQuery } from '@apollo/react-hooks';
+
+import * as Style from './style';
+
+import { SearchRoomContext } from 'pages/SearchRoom';
+
+import RoomItem from 'components/RoomItem';
+import Modal from 'components/Modal';
```

Comment:
```suggestion
import { RoomItem, Modal } from 'components';
```
이와 같이 import 할 수 있도록 `react-app/src/components` 하단에 index.js 파일에서 `components/RoomItem`, `components/Modal`를 import 해오는 방식은 어떠신가요?